validation/src/main/scala/hmda/validation/rules/lar/quality/Q066.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.validation.rules.lar.quality
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.{Failure, Result}
+import hmda.validation.rules.EditCheck
+
+import scala.util.Try
+
+object Q066 extends EditCheck[LoanApplicationRegister] {
+  override def apply(lar: LoanApplicationRegister): Result = {
+    val config = ConfigFactory.load()
+    val rateSpread = config.getDouble("hmda.validation.quality.Q066.rate-spread")
+
+    when(lar.rateSpread not equalTo("NA")) {
+      Try(lar.rateSpread.toDouble is lessThan(rateSpread))
+        .getOrElse(Failure(s"Can't parse '${lar.rateSpread}' as a Double"))
```

Comment:
This seems like logic that could live in the DSL, analogous to `numericallyBetween` (and yes, probably being consistent with it in using whichever of BigDecimal or Double is deemed appropriate).